helper-scripts/beautify-schedule.ts

```diff
@@ -230,8 +230,8 @@ async function convertTeamworkHandlesToZulipHandles(shiftsJson: any) {
 			const agentHandleAndEmail = shift.agent;
 			const emailRegex = /<([^<>]+)>/;
 			const matches = agentHandleAndEmail.match(emailRegex);
-			const email = matches ? matches[1] : null;
-			if (email in usersByEmail) {
+			const email = matches ? matches[1].toLowerCase() : null;
+			if (email && email in usersByEmail) {
```

Comment:
`if (email)` is effectively a newly introduced check. I don't have extensive JS/TS knowledge but don't think a dictionary key can technically be `NULL`. The performance gains must be negligible for a hash lookup. Do you see other notable benefits?

Reply:
You are right. It is just playing it safe here.